Build Failure in /src/pyodide-build

I just made a fresh clone of the `main` branch and am trying to follow the build instructions for the Docker container. I run `make` inside the docker container and it will build the `.wasm` file for Python but not the standard library. I get the following error:

```
make[1]: Entering directory '/src'
Ensuring pyodide-build is installed
pip install -e ./pyodide-build
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///src/pyodide-build
ERROR: file:///src/pyodide-build does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
make[1]: *** [Makefile:229: pyodide_build] Error 1
```

--- a/docs/development/building-from-sources.md
+++ b/docs/development/building-from-sources.md
@@ -24,6 +24,19 @@ Pyodide can be built from sources on different platforms,
 
 ## Build instructions
 
+The Pyodide repository has a git submodule called `pyodide-build`. Make sure to
+do a recursive clone:
+```bash
+git clone --recursive https://github.com/pyodide/pyodide
+```
+or if you have already cloned the Pyodide repository without the `--recursive`
+flag, you may initialize the submodule with:
+```bash
+git submodule update --init
+```
+If you change git branches, make sure you update `pyodide-build` with
+`git submodule update`.
+
 ### Using Docker
 
 We provide a Debian-based x86_64 Docker image

--- a/docs/development/building-from-sources.md
+++ b/docs/development/building-from-sources.md
@@ -26,14 +26,18 @@ Pyodide can be built from sources on different platforms,
 
 The Pyodide repository has a git submodule called `pyodide-build`. Make sure to
 do a recursive clone:
+
 ```bash
 git clone --recursive https://github.com/pyodide/pyodide
 ```
+
 or if you have already cloned the Pyodide repository without the `--recursive`
 flag, you may initialize the submodule with:
+
 ```bash
 git submodule update --init
 ```
+
 If you change git branches, make sure you update `pyodide-build` with
 `git submodule update`.
 

